cmd/oklog/forward_slicebuffer.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"sync"
+)
+
+// sliceBuffer is a fixed-length buffer. It can be used by a forwarder, to 'drop' messages instead of applying backpressure. See Issue #15
+type sliceBuffer struct {
+	//onward io.Writer
+	//prefix string
+	max int
+
+	buf   []string
+	ch    chan string
+	mutex sync.RWMutex
+}
+
+// Put forwards right away when data is needed
+func (b *sliceBuffer) Put(record string) {
+	select {
+	case b.ch <- record:
+	default:
```

Comment:
I've used the select/default on the send side only, for one-way blocking. We always want Get to wait for data, but we want Put to buffer instead.